plugins/copr.py

```diff
@@ -256,8 +260,8 @@ def run(self):
 Please do not file bug reports about these packages in Fedora
 Bugzilla. In case of problems, contact the owner of this repository.
 
-Do you really want to enable {0}?""".format('/'.join([self.copr_hostname,
-                                                      copr_username, copr_projectname])))
+""").format('/'.join([self.copr_hostname, copr_username, copr_projectname])))
+            msg = "Do you really want to enable {0}?".format(self.copr_hostname)
```

Comment:
This new prompt only prints the `copr_hostname`, the previous one had the whole `hostname/username/projectname` identifier. Note this prompt is not tested by the tests...

Reply:
Fixed.